website/docs/error-reference.md

```diff
@@ -702,6 +702,68 @@ you'll get this error—the latter export is redundant, as it's implied by expor
 
 To fix this error, simply remove the more specific export.
 
+## 3717
+
+> This error is specific to Stripe's custom `--stripe-packages` definition. If
+> you are at Stripe, please see [go/modularity](http://go/modularity) for more.
+
+This error means that you're using a name that's private in another package, and
+not epxorted.
```

Comment:
Is it private **and** not exported?

Stripe's modularity guidelines also have rules about `Private` classes, and I think that the `Private` namespace concept is separate.

Also a question:

Does this error only happen when the package containing the definition **is** imported in this package, or could it ever be the case that the user would have to **both** import the package in this package **and** export the relevant definition in the upstream package?

Could we capture the answer in the docs here?

Reply:
We can have both situations arise, though they would get different errors associated with them. I can clarify this a bit here.